venues/aclweb.org/ACL/ethicsReviewersWebfield.js

```diff
@@ -0,0 +1,461 @@
+// webfield_template
+// Remove line above if you don't want this page to be overwriten
```

Comment:
I copied this into the aclweb.org/ACL/2022/Conference/Ethics_Reviewers UI code but got 'Unhandled Runtime Error
TypeError: Cannot read property 'user' of undefined' when trying to view it as super user. (This was also after adding a tilde ID and a PaperX/Ethics_Reviewers group to the aclweb.org/ACL/2022/Conference/Ethics_Reviewers group)

Reply:
can you impersonate the tilde ID and go to the console?

Reply:
I get error 403

Reply:
the member of the group is not reader of the group, you can fix it here: https://github.com/openreview/openreview-scripts/pull/1231/files#r798922788